MediumProject/src/main/java/org/example/stage2/Searching.java

```diff
@@ -0,0 +1,68 @@
+package org.example.stage2;
+
+import java.util.*;
+
+public class Searching {
+
+    private static List<String[]> listOfData(){
+        System.out.println("Enter the number of people:");
+        Scanner scan = new Scanner(System.in);
+        int size = scan.nextInt();
+        scan.nextLine();
+        System.out.println("Enter all people:");
+        List<String[]> data= new ArrayList<>();
+        String[] input = new String[size];
+        for (int i = 0; i < input.length; i++) {
+            input[i] = scan.nextLine();
+        }
+        System.out.println();
+        for (int i = 0; i < size; i++) {
+            data.add(input[i].split(" "));
```

Comment:
convert array in a list and later use list.containsIgnoseCase util method to search word